pages/en/guides/index.md

```diff
@@ -22,6 +22,6 @@ layout: docs.hbs
 
 ## Module-related guides
 
-- [Working with Different Filesystems](/guides/working-with-different-filesystems/)
+- [Anatomy of an HTTP Transaction](/guides/anatomy-of-an-http-transaction/)
```

Comment:
https://github.com/nodejs/nodejs.org/pull/6256 moved this out of Guides to the Learn section so I don't think it should be listed here?

Reply:
done via https://github.com/nodejs/nodejs.org/pull/6267/commits/21c77ec637355e216f9d0ca61ffb59dea6caad2d